app/app.web.view.ts

```diff
@@ -15,8 +15,9 @@ namespace $.$$ {
 
 		@ $mol_mem
 		pages(): readonly any[] {
-			const in_iframe = window.self !== window.top
-			
+			// hide menu being embedded but show upon ?menu
+			const in_iframe = (window.self !== window.top && document.location.search.indexOf('menu') == -1)
```

Comment:
it can be done with `$mol_state_arg.value`
```suggestion
			const in_iframe = (window.self !== window.top) && (this.$.$mol_state_arg.value('menu') == null)
```
`this.$.$mol_state_arg.value('menu')` is empty string for https://nanoshow.mpds.io/#!menu and is `null` for https://nanoshow.mpds.io

`$mol_state_arg.value` is better, because we can change and read link params without manual parsing

Reply:
Thanks, it's interesting, but the app hash can be used for something else in future (cf. https://github.com/tilde-lab/cifplayer/issues/22), so here's my motivation to use `location.search`.